content/Foundation/Summit/home.pml

```diff
@@ -0,0 +1,90 @@
+<!--<title>Summit</title>-->
+<!--
+    Copyright 2020 seL4 Project a Series of LF Projects, LLC.
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+<h1>
+  seL4 Summit 2022: Call for Bids
+</h1>
+<div>
+  <p>
+   The seL4 Summit 2022 will be the fourth edition of the international summit on the seL4 microkernel, the world's most highly assured OS kernel.
+  </p>
+  <p>
+   <strong>We are now calling for bids to host the seL4 Summit 2022.</strong>
+  </p>
+  <p>
+  <strong>Bids should be sent to  <a title="summit@sel4.systems"   href="mailto:summit@sel4.systems">summit@sel4.systems</a> before 22 February 2022.</strong>
+  </p>
+
+  <h3>
+   Background
+  </h3>
+  <p>
+   The seL4 Summit 2022 is the first to be organised by the seL4 Foundation. Established in 2020, the seL4 Foundation brings together developers of the seL4 kernel, developers of seL4-based components and frameworks, and those adopting seL4 in real-world systems.
+  </p>
+  <p>
+   The first three seL4 Summits were organised by the <a href="https://trustedcomputingcoe.org/summits/">Trusted Computed Center of Excellence</a>, and were hosted in the US.
+  </p>
+  <p>
+   The seL4 Foundation now aims to host the summit on a different continent each year, as far as possible. For 2022, we therefore solicit bids from prospective hosts with a preference for location outside of North America.
+  </p>
+
+  <h3>
+   Principles and process
+  </h3>
+  <p>
+   There will be two separate bodies in charge of the summit: a Program Committee, in charge of the technical side (content, invited speakers, program...) and an Hosting Team, in charge of organising the event (venue, finances, catering, local arrangements...).
+  </p>
+  <p>
+   More information on the Program Committee will come soon.
+  </p>
+  <p>
+   The Hosting Team will be selected through an open bid process by the seL4 Foundation board, after consultation with the community.
+  </p>
+  <p>
+   In the future instances, the call for bids will ideally occur 2 months before the previous summit, in order for the bids for year N+1 to be presented at the summit in year N. As the 2022 summit is the first to be organised by the seL4 Foundation, the <strong>call for bids are done now, and will close on 22 February 2022</strong>.
+  </p>
+  <p>
+   We would like to aim for a in-person event, with the option of recordings of the talks being available after the summit. A back-up plan for a virtual or hybrid event in case of pandemic-related travel restrictions should be defined as part of the bid application.
```

Comment:
It seems unlikely that there will be no travel restrictions at all by Oct, so the default should probably be hybrid.

Maybe:

We would like to aim for a hybrid in-person/online event, pre-dominantly in-person, with the option of recordings of the talks being available after the summit. A back-up plan for a virtual event in case of pandemic-related travel restrictions should be defined as part of the bid application.